### DISCUSSION TITLE
ring-mqtt goes offline

### DISCUSSION THREAD

**Discussion by agecrabahaykubo** [tsightler/ring-mqtt#301] [Support Q&A]
First of all thank you very much for your work on this awesome home automation integration. I really liked how easy to implement  automation to my ring alarm devices ONLY if it stays ONLINE. But it goes offline after some time and I needed to reboot my system for it to work again.

When I check the service status it gives the error CODE EXITED, 209=STDOUT. When I run the debug command the device status shows as ONLINE but it is not really "working" like no motion is detected on my motion detector sensor.

I installed ring-mqtt as STANDARD INSTALL on RPi4 (OpenHAB).

**Comment by tsightler** [comment]
Without debug logs there's really nothing to go on here.  The 209=STDOUT error is usually related to systemd, not to the code itself.  If you can provide debug logs that show that ring-mqtt is in some way malfunctioning, I can try to help, but, other than that, when you use the standard install method, you are largely on your own which is why the Docker method is the recommended method.
